src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working.");

```

Comment:
Similarly, the stop message should also be informative. It could be rephrased to 'Excavator has stopped working.' to maintain consistency and clarity.